.github/workflows/test.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           - macos-12
           - windows-2019
         cmake-version:
+          - 3.31.0
           - 3.20.0
```

Comment:
This should also probably be set to 3.22.1 which is the version in ubuntu 22.04.